vedtak/application/src/main/kotlin/no/nav/su/se/bakover/vedtak/application/VedtakService.kt

```diff
@@ -30,4 +34,9 @@ interface VedtakService {
     fun hentForUtbetaling(utbetalingId: UUID30): VedtakSomKanRevurderes?
     fun hentForFødselsnumreOgFraOgMedMåned(fødselsnumre: List<Fnr>, fraOgMed: Måned): List<Vedtaksammendrag>
     fun hentSøknadsbehandlingsvedtakFraOgMed(fraOgMed: LocalDate): List<UUID>
+
+    /**
+     * Per nå, så kan bare vedtak som er blitt avslått av §18 (manglende dokumentasjon) åpne en ny behandling.
```

Comment:
John Are ønsket ikke dette. De kunne f.eks. avslå pga. formue for deretter motta informasjon innen 6 uker og deretter måtte innvilge.

Reply:
ja.. gammel kommentar før vi hadde den samtalen